embc-app/DataInterfaces/DataInterface.EvacueeRegistration.cs

```diff
@@ -36,6 +38,9 @@ public async Task<Registration> CreateEvacueeRegistrationAsync(Registration regi
         public async Task UpdateEvacueeRegistrationAsync(Registration registration)
         {
             var evacueeRegistration = registration.ToModel();
+
+            evacueeRegistration.CompletedById = httpContextAccessor?.HttpContext?.User?.FindFirstValue(EssClaimTypes.USER_ID);
```

Comment:
Can you please move this to the controller instead of dragging the dependency all the way down?

Reply:
Yeah, much better plan.

Reply:
Moved the logic to the controller